firmware/config/boards/Benelli_Walbro/connectors/Benelli_Walbro.yaml

```diff
@@ -230,34 +233,38 @@ pins:
 
   - pin: F1
     function: Power Ground
-    class: gnd
-    type: gnd
+    class: pgnd
+    type: pgnd
     color: black
 
   - pin: F2
     # not required by rusEFI??
     function: 'OBD2 - DIAG RS232 Tx/Rx'
+    class: OBD2
 
   - pin: F3
     # not required by rusEFI?? (OBD2?)
-    function: 'OBD2 - K line)'
+    function: 'OBD2 - K-line'
+    class: OBD2
 
   - pin: F4
     # not required by rusEFI?? (OBD2?)
-    function: 'OBD2- L Line)'
+    function: 'OBD2- L-Line'
+    class: OBD2
 
   - pin: F5
     function: 'Tilt Sensor (Grounded or Open circuit)'
+    class: input_ON_OFF
 
   - pin: F6
-    function: side stand Sensor Input  (grounded or open circuit)
-    class: analog_inputs
+    function: 'side stand Sensor Input  (grounded or open circuit)'
+    class: input_ON_OFF
     color: white
 
   - pin: F7
     function: 'Injection warning light (MIL)'
-    class: analog_inputs
-    type: ??
+    class: analog_outputs
```

Comment:
please clarify what is ``analog_outputs``?

see https://github.com/rusefi/rusefi/blob/master/.github/workflows/gen-pinouts.yaml for all official types. MIL is not one of those types. Happy to merge as is just need your awareness.

Reply:
Sorry, can't seem to find "official types" in the referenced file :( 
Had already created https://github.com/chuckwagoncomputing/interactive-pinout/issues/16 to have a documented list for reference.
Any hint, how I would code MIL correctly?

Reply:
@mi-hol please note lines 23 and below

Reply:
Sorry, but even with this hint, I fail to see the list of valid "class" values to fix "class: analog_outputs".

1. "class" is not mentioned in
`        columns: |
          {
          "pin":"Pin Number",
          "ts_name":"TS Name",
          "type":"Type",
          "function":"Typical Function",
          "color":"Pigtail Color"
          }
`
Therefore to fix "class: analog_outputs"  I'm still unclear about "how to code MIL correctly"

2. the list of valid "type" values seems to start at line 44 but is hard to digest for an inexperienced user due to the preceeding "color" label. Maybe a simple comment would avoid this hurdle for now?

![image](https://github.com/rusefi/rusefi/assets/22799428/76b4264c-3314-42a2-b57e-028e97567b7c)